src/core/server/saved_objects/migrationsv2/integration_tests/migration.test.ts

```diff
@@ -215,27 +242,34 @@ describe('migration v2', () => {
         },
         { ignore: [404] }
       );
-
       const response = body[migratedIndex];
 
-      expect(response).toBeDefined();
-      expect(Object.keys(response.aliases!).sort()).toEqual([
-        '.kibana',
-        `.kibana_${kibanaVersion}`,
-      ]);
+      if (response && response !== null) {
```

Comment:
When/why would the `response` be `null` or `undefined`?

Reply:
Oops, that's left over from debugging.